src/views/HelpAndKnowledge/Articles/SixPagerMeeting/index.tsx

```diff
@@ -0,0 +1,564 @@
+import UswdsReactLink from 'components/LinkWrapper';
+import MainContent from 'components/MainContent';
+import PageHeading from 'components/PageHeading';
+import RelatedArticles from 'components/RelatedArticles';
+import { findSolutionByRouteParam } from 'views/HelpAndKnowledge/SolutionsHelp';
+import SolutionDetailsModal from 'views/HelpAndKnowledge/SolutionsHelp/SolutionDetails/Modal';
+import { helpSolutions } from 'views/HelpAndKnowledge/SolutionsHelp/solutionsMap';
+
+const covertToLowercaseAndDashes = (string: string) =>
+  string.toLowerCase().replace(/\s+/g, '-');
+
+const Link = ({ scrollTo }: { scrollTo: string }) => {
+  return (
+    <a
+      href={`#${covertToLowercaseAndDashes(scrollTo)}`}
+      className="display-flex flex-align-center"
+    >
+      {scrollTo}
+      <IconArrowForward />
+    </a>
+  );
+};
+
+const SixPagerMeeting = () => {
+  const { t: sixPageMeetingT } = useTranslation('sixPageMeeting');
+  const [LDGmodal, setLDGmodal] = useState(false);
+
+  const modelOverviewAndGoals: string[] = sixPageMeetingT(
+    'conceptPaper.stepOne.items',
+    { returnObjects: true }
+  );
+
+  const overviewOfKeyModelDesignElements: string[] = sixPageMeetingT(
+    'conceptPaper.stepTwo.subitems',
+    { returnObjects: true }
+  );
+
+  const alignmentListItems: string[] = sixPageMeetingT(
+    'conceptPaper.stepThree.alignment.numberedList',
+    { returnObjects: true }
+  );
+
+  const impactListItems: string[] = sixPageMeetingT(
+    'conceptPaper.stepThree.impact.list',
+    { returnObjects: true }
+  );
+
+  const stepFour: string[] = sixPageMeetingT('conceptPaper.stepFour.items', {
+    returnObjects: true
+  });
+
+  const stepFive: string[] = sixPageMeetingT('conceptPaper.stepFive.items', {
+    returnObjects: true
+  });
+  const stepSix: string[] = sixPageMeetingT('conceptPaper.stepSix.items', {
+    returnObjects: true
+  });
+  const stepSeven: string[] = sixPageMeetingT('conceptPaper.stepSeven.items', {
+    returnObjects: true
+  });
+
+  const additionalResourcesListItems: string[] = sixPageMeetingT(
+    'additionalResources.list',
+    { returnObjects: true }
+  );
+  const additionalResourcesParagraphs: string[] = sixPageMeetingT(
+    'additionalResources.paragraph',
+    { returnObjects: true }
+  );
+  const tipsList: string[] = sixPageMeetingT(
+    'reviewMeeting.subheading.tip.list',
+    { returnObjects: true }
+  );
+  const outcomesParagraphs: string[] = sixPageMeetingT(
+    'reviewMeeting.subheading.outcomes.paragraph',
+    { returnObjects: true }
+  );
+
+  const selectedSolution = findSolutionByRouteParam(
+    'learning-and-diffusion-group',
+    helpSolutions
+  );
+
+  return (
+    <>
+      {LDGmodal && selectedSolution && (
+        <SolutionDetailsModal
+          solution={selectedSolution}
+          openedFrom="help-article"
+          closeRoute="/help-and-knowledge/how-to-have-a-successful-6-pager-meeting"
+          setLDGmodal={setLDGmodal}
+        />
+      )}
```

Comment:
This method of rendering the modal doesn't work as expected/how it was built.  The url get appended with params and never clears them, also on close the page pops back up.  You can reference how this was implemented with IT Solution Tracker.  It can be implemented the same way here.  I was able to deprecate the `setLDG` modal in favor of this:

```
const location = useLocation();

  const [initLocation] = useState<string>(location.pathname);

  const { prevPathname, selectedSolution, renderModal } = useModalSolutionState(
    OperationalSolutionKey.LDG
  );

  const ldgRoute = `${initLocation}${location.search}${
    location.search ? '&' : '?'
  }solution=learning-and-diffusion-group&section=about`;
```

Then for the modal it looks like this:
```
{renderModal && selectedSolution && (
        <SolutionDetailsModal
          solution={selectedSolution}
          openedFrom={prevPathname}
          closeRoute="/help-and-knowledge/how-to-have-a-successful-6-pager-meeting"
        />
      )}
```

The button to open then becomes a link :
```
<UswdsReactLink
                      className="display-flex flex-align-center usa-button usa-button--unstyled margin-top-2"
                      to={ldgRoute}
                    >
                      {sixPageMeetingT('crossCuttingGroupsSummaryBox.item.ldg')}
                      <IconArrowForward className="margin-left-1" />
                    </UswdsReactLink>
```

Because the modal was built with internal navigation, it's implementation should follow that paradigm.  The hook above returns everything to manage routes/scrolling with the model component.  If you want let's sync up, would be happy to clear up anything